src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,15 +1,101 @@
-import { Component } from 'react';
+import React from 'react';
 
-type Props = {};
-type State = {};
+import './NewMovie.scss';
 
-export class NewMovie extends Component<Props, State> {
-  state: State = {};
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+type State = {
+  title: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+  description: string;
+};
+
+export class NewMovie extends React.Component<Props, State> {
+  state: State = {
+    title: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+    description: '',
+  };
+
+  createMovie = (props: Movie) => (
```

Comment:
do not call it `props`, it's confusing
```suggestion
  createMovie = (movie: Movie) => (
```